Libraries/LibWeb/Layout/TreeBuilder.cpp

```diff
@@ -363,6 +373,9 @@ void TreeBuilder::create_pseudo_element_if_needed(DOM::Element& element, CSS::Ps
                 insert_node_into_inline_or_block_ancestor(*layout_node, layout_node->display(), AppendOrPrepend::Append);
             }
             pop_parent();
+
+            if (needs_reversed_counter_fixup)
+                ensure_content_with_reverse_counter_fixup_queue().append({ pseudo_element_reference, move(text_nodes_needing_fixup), initial_quote_nesting_level });
```

Comment:
This is the only place a `PseudoElementWithReversedCounterContent` is created. `text_nodes_needing_fixup` isn't null, because of lines 351 and 352: `needs_reversed_counter_fixup` is true.

So, there is no state where a `PseudoElementWithReversedCounterContent` exists and its `list_of_text_nodes` is null. Unless I'm still missing something?